store/action-queue.html

```diff
@@ -21,7 +21,7 @@
 					}
 				};
 			});
-			this.queueEnd.then(taskRunner).catch(taskRunner);
+			this.queueEnd.then(taskRunner, taskRunner);
```

Comment:
I spent a while trying to write a test showing this could run twice, mostly because I was thinking of the Promise rejecting and not the inner task runner itself.

This should be more obvious that it won't, and also allocates one less Promise.